docs/docsite/rst/playbook_guide/playbooks_reuse_roles.rst

```diff
@@ -348,7 +370,7 @@ role ``meta/argument_specs.yml`` file. All fields are lowercase.
 
             * If ``required`` is ``false``/missing, ``default`` may be specified (assumed ``null`` if missing).
             * Ensure that the default value in the docs matches the default value in the code. The actual
-              default for the role variable will always come from ``defaults/main.yml``.
+              default for the role variable will always come from the ``defaults/main.yml`` file (also ``main.yaml``, ``main`` or 'specified'). To learn more about these options see :ref:`Role directory structure <role_directory_structure>`.
```

Comment:
I think it could be pretty useful to see the file path and name right away, instead of having to navigate to a different section. Maybe I would just rephrase it a bit, or remove the 'specified' option, as it is relevant only under certain circumstances

Reply:
then i would make notes/refs with every inclusion of the 'default path' and explain that though we show only one version, many are possible.

Reply:
I'm not sure what exactly you mean by that. I thought the possible versions are main.yml/main.yaml/main (or specified)

Reply:
To explain @bcoca's comment - I believe he is saying if we spell out every options here, we should spell it out everywhere we talk about defaults.

In this instance, though, I agree with bcoca's [initial comment](https://github.com/ansible/ansible-documentation/pull/1320/files#r1586403867) - let's not spell them all out here. While I would in general agree that having short information like this handy, there are two reasons I think we should not do it in this case:
1 - This is a piece of reference documentation, so being more verbose is not necessarily helpful.
2 - If this changes in the future, we have more places to update the content. So having one place where it's defined makes things easier to maintain.